README.md

```diff
@@ -1 +1,104 @@
-# megafuel-js-sdk
\ No newline at end of file
+# megafuel-js-sdk
+
+This JavaScript SDK is thin wrapper of MegaFuel clients, offering a streamlined interface to interact with the MegaFuel. For more information, please refer to the [API documentation](https://docs.nodereal.io/docs/megafuel-api).
+
+## Network Endpoint
+
+- BSC mainet: https://bsc-megafuel.nodereal.io
+- BSC testnet: https://bsc-megafuel-testnet.nodereal.io
+-
```

Comment:
typo `-`? 

can you also list the sponsor endpoint? 
`https://open-platform.nodereal.io/<api-key>/megafuel-testnet`
`https://open-platform.nodereal.io/<api-key>/megafuel`